server.go

```diff
@@ -93,6 +99,10 @@ type Server struct {
 	// is just getting the request after being off duty for running CQs then
 	// it will recompute all of them
 	lastContinuousQueryRun time.Time
+
+	// Build information.
```

Comment:
I think it's important the server object get this stuff, so it be dumped by the diags. I am open to other approaches.